tests/app/questionnaire/test_placeholder_parser.py

```diff
@@ -939,7 +945,7 @@ def test_placeholder_default_value(default_placeholder_value_schema, mock_render
         language="en",
         answer_store=AnswerStore(),
         list_store=ListStore(),
-        metadata={},
+        metadata=get_metadata(),
```

Comment:
👍 This is what you should do in `test_questionnaire_form.py`

Reply:
Refactored this file now